units_cpp/game_state.h

```diff
@@ -1,42 +1,56 @@
+/* Defines the class GameState, which contains all information relevant to a single Adrestia game. */
+
```

Comment:
I disagree with two blank lines above the #pragma once (really, any sort of double-blank-lining), though getting into a fight about it will probably cause more productivity loss than simply acquiescing to your taste in code style.